### PR TITLE
MAUI-4761 : Update UG for MouseHover support in .NET MAUI ListView

### PR DESCRIPTION
**Pull request by RiyashameedM**
Update UG for MouseHover support in .NET MAUI ListView

### CODE REVIEW

File: MAUI/ListView/viewappearance.md [hunk 1]
```diff
@@ -499,6 +499,116 @@ public partial class MainPage : ContentPage
 
 ![MAUI ListView Drop shadow effect on items](Images/appearance/maui-listview-drop-shadow-effects-on-items.jpg)
 
+## Visual State Manager
+
+Use the visual state manager to change the .NET MAUI SfListView mouse hover background on the listView item based on the visual states set from code. The applicable visual states are Normal and PointerOver.
```
Comment (florenceliliana) [inline MAUI/ListView/viewappearance.md:504]: Use the visual state manager to change the .NET MAUI SfListView mouse hover background on the listView item based on the visual states set from the code. The applicable visual states are Normal and PointerOver.

**Review by florenceliliana — CHANGES_REQUESTED** [review]

File: MAUI/ListView/viewappearance.md (hunk 1 above)
Comment (RiyashameedM) [reply MAUI/ListView/viewappearance.md]: resolved

File: MAUI/ListView/viewappearance.md [hunk 2]
```diff
@@ -499,6 +499,49 @@ public partial class MainPage : ContentPage
...
+    </ContentPage.Resources>
+    <ContentPage.Content>
+        <syncfusion:SfListView ItemsSource="{Binding ContactsInfo}">
+        </syncfusion:SfListView>
+    </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% endtabs %}
+
+N> Only the `Background` property of the visual state can be customized.
```
Comment (PiruthivirajM) [inline MAUI/ListView/viewappearance.md:543]: Only the Background property of ListViewItem can be customized using visual state.

File: MAUI/ListView/viewappearance.md [hunk 3]
```diff
@@ -499,6 +499,49 @@ public partial class MainPage : ContentPage
 
 ![MAUI ListView Drop shadow effect on items](Images/appearance/maui-listview-drop-shadow-effects-on-items.jpg)
 
+## Change mouse hover item background
+
+Use the visual state manager to change the `SfListView` mouse hover background on the `ListViewItem` based on the visual states set from the code. The applicable visual states are `Normal` and `PointerOver`.
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/ListView/viewappearance.md]: resolved

**Pull request merged**